pygeos/test/test_creation.py

```diff
@@ -318,3 +318,26 @@ def test_subclasses(with_point_in_registry):
         assert isinstance(point, Point)
         assert pygeos.get_type_id(point) == pygeos.GeometryType.POINT
         assert point.x == 1
+
+
+def test_prepare():
+    arr = np.array([pygeos.points(1, 1), None, pygeos.box(0, 0, 1, 1)])
+    pygeos.prepare(arr)
```

Comment:
Before `prepare` it is probably a good idea to confirm that `_ptr_prepared` is in default state:

```
assert arr[0]._ptr_prepared == 0
assert arr[2]._ptr_prepared == 0
```